src/config/routes.jsx

```diff
@@ -0,0 +1,21 @@
+import { BrowserRouter, Route, Switch } from 'react-router-dom';
+import Shop from '../components/Shop';
+import MyCart from '../components/MyCart';
+import Checkout from '../components/Checkout';
+import OrderConf from '../components/OrderConf';
+
+const Routes = (props) => {
+  return (
+    <BrowserRouter>
+      <Switch>
+        <Route exact path="/shop"><Shop/></Route>
+        <Route path="/cart"><MyCart/></Route>
+        <Route path="/checkout"><Checkout/></Route>
+        <Route path="/order-conf"><OrderConf/></Route>
```

Comment:
Just as an aside there are a few different ways you can define a Route within a Route component.
```<Route path="/shop" component={Shop} />``` ← Typical use case
```<Route path="/shop" render={() => <Shop />} />``` ← If you need to pass props
```<Route path="/shop"><Shop></Route>``` ← My less preferred use case